_posts/2022-03-23-#21778.md

```diff
@@ -10,14 +10,39 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+Historically Bitcoin Core has used a very ["non-standard" toolchain](https://github.com/bitcoin/bitcoin/blob/master/contrib/macdeploy/README.md) for producing its macOS release binaries. This is primarily due to there not being any "official" way to build macOS binaries, using completely open source software, on a Linux system. The only Apple-sanctioned way to build macOS binaries is using the Xcode toolchain, running on macOS hardware.
```

Comment:
```suggestion
Historically, Bitcoin Core has used a very ["non-standard" toolchain](https://github.com/bitcoin/bitcoin/blob/master/contrib/macdeploy/README.md) for producing its macOS release binaries. This is primarily due to there not being any "official" way to build macOS binaries, using completely open source software, on a Linux system. The only Apple-sanctioned way to build macOS binaries is using the Xcode toolchain, running on macOS hardware.
```

Reply:
You may want to clarify here that you're talking about building for macOS on a linux machine, i.e. you're not just talking about building bitcoin core on a macOS machine. I don't think it's immediately obvious. You might even want to start by saying Bitcoin Core releases binaries for multiple platforms and link to https://en.wikipedia.org/wiki/Cross_compiler, just to give someone a ramp if they're starting from 0.

Reply:
Added something here.